src/Honeybee_Generator_Wind_Horizontialaxis.py

```diff
@@ -134,206 +134,212 @@ def checktheinputs(_name,_simpleOrAnalytical,_powerCoefficients,_powerControl,_r
         ghenv.Component.AddRuntimeMessage(w, "Only one template can be used at a time! Please set either TemplateLargeTurbine_ or TemplateMediumTurbine_ to true not both!")
         return -1
 
-    if _powerControl == None:
-        print "The field _powerControl must be specified!"
+    if powerControl_ == None:
```

Comment:
@mostaphaRoudsari 
Sorry the delay I lost my Grasshopper for about 10 days (was getting withdrawal symptoms). They are all optional inputs hence I had to change them. Are you able to merge this pull request now?